.github/workflows/continuous-integration.yml

```
@@ -379,18 +378,15 @@ jobs:
             .cache/yarn
             node_modules
 
-      - name: Set IS_MASTER_BUILD
+      - name: List changed files
+        id: changed-files
         run: |
           if [[ ${{ github.ref }} == 'refs/heads/master' ]]; then
-              echo "IS_MASTER_BUILD=true" >> $GITHUB_ENV
+            echo ::set-output name=all_changed_files::$(git diff-tree --no-commit-id --name-only -r ${{ commit_sha }})
```

```suggestion
            echo ::set-output name=all_changed_files::$(git diff-tree --no-commit-id --name-only -r ${{ github.sha }})
```
Since `commit_sha` isn't defined here